surveys/2024-04/questions.md

```diff
@@ -0,0 +1,160 @@
+# Survey April 2024
+
+## Demographics
+
+- Where are you from? (countries list if possible - or open question)
+  - country
+
+- How long have you been working with Node.js? (numbers list from 0 to 10+)
+
+- In what kind of organizations are you working in?
+  - big tech
+  - company
+  - startup
+  - university
+  - individual
+  - other
+
+## Node.js Usage
+
+- Which groups do you identify with? (multiple choice)
+  - Direct end users: Users who run tools themselves
+  - Application operators: Users who interacts with existing running applications
+  - Application developers: Frontend tools consumers, backend server authors, tools authors
+  - Library & package authors: Users who write libraries and packages to be included on other applications
+  - Node.js core maintainers: Developers working directly on Node.js, individuals participating in Working Groups
+  - Organization with investments in Node.js (eg: Enterprises, Gouvernement bodies, Startups, Non-profit, Education, Security)
+
+- Are you part of a group not covered ? If yes which one ? (open question)
+
+- What is your use cases of Node.js ? (if multiple use case, separate with a comma) (open question)
+
+## What Node.js binaries do you use
+
+- What is your Operating System for your local development environment ? (single choice - Other should give an input)
+  - Linux
+  - Windows
+  - macOS
+  - Other?
+
+- What is your Operating System in which you are running Node.js in production ? Do you plan to change that in the near future? (single choice - Other should give an input)
```

Comment:
"Do you plan to change" -> how can we get the information if people answer with a checkbox?